crates/shared/src/price_estimation/trade_finder.rs

```diff
@@ -164,7 +164,20 @@ impl TradeVerifier {
             },
         };
 
-        let return_data = self.simulator.simulate(call, overrides).await?;
+        let return_data = match self.simulator.simulate(call, overrides).await {
+            Ok(data) => data,
+            Err(SimulationError::Other(err)) => {
+                // In case we have a simulator error (network, service is down,
```

Comment:
Comment make sense. But the last sentence is not complete

Reply:
🙈